src/platforms/javascript/guides/sveltekit/manual-setup.mdx

```diff
@@ -455,3 +455,48 @@ export const load = wrapServerLoadWithSentry((event) => {
   //... your load code
 });
 ```
+
+### Configure Client-side `fetch` Instrumentation
+
+<Note>
+
+Available since version 7.91.0
+
+</Note>
+
+The `sentryHandle` function you added to your `handle` hook in `hooks.server.ts` during [server-side setup](#server-side-setup) injects a small inline `<script>` tag into the HTML response of the server.
+This script attempts to proxy all client-side `fetch` calls, so that `fetch` calls inside your `load` functions are captured by the SDK.
+However, if you configured CSP rules to block inline fetch scripts by default, this script will be [blocked by the browser](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#unsafe_inline_script).
+To enable the script, you need to add an exception for the `sentryHandle` script:
+
+First, specify a nonce in the `fetchProxyScriptNonce` option in your `sentryHandle` call:
+
+```javascript {filename:hooks.server.ts}
+// Add the nonce to the <script> tag:
+export const handle = sentryHandle({ fetchProxyScriptNonce: "2726c7f26c" });
```

Comment:
Is this the actual nonce users need to define or do they potentially have a different one?

Reply:
Ahh good point, this might be missleading. No, it can be any nonce. Just needs to match the one specified in the CSP. Probably better to replace with `fetchProxyScriptNonce: "_YOUR_NONCE_ `